cumulus/zombienet/zombienet-sdk/tests/zombie_ci/elastic_scaling_slot_based_authoring.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+use std::time::Duration;
+
+use crate::utils::{initialize_network, wait_node_is_up, BEST_BLOCK_METRIC};
+
+use cumulus_zombienet_sdk_helpers::{
+	create_assign_core_call, submit_extrinsic_and_wait_for_finalization_success_with_timeout,
+};
+use serde_json::json;
+use zombienet_orchestrator::network::node::LogLineCountOptions;
+use zombienet_sdk::{
+	subxt::{OnlineClient, PolkadotConfig},
+	subxt_signer::sr25519::dev,
+	NetworkConfig, NetworkConfigBuilder,
+};
+
+const PARA_ID_1: u32 = 2100;
+const PARA_ID_2: u32 = 2000;
+
+// TODO
+#[tokio::test(flavor = "multi_thread")]
+async fn elastic_scaling_slot_based_authoring() -> Result<(), anyhow::Error> {
+	let _ = env_logger::try_init_from_env(
+		env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+	);
+
+	log::info!("Spawning network");
+	let config = build_network_config().await?;
+	let network = initialize_network(config).await?;
+
+	let alice = network.get_node("alice")?;
+	let collator_elastic = network.get_node("collator-elastic")?;
+	let collator_single_core = network.get_node("collator-single-core")?;
+
+	log::info!("Checking if alice is up");
+	assert!(wait_node_is_up(alice, 60u64).await.is_ok());
+
+	log::info!("Checking if collator-elastic is up");
+	assert!(wait_node_is_up(collator_elastic, 60u64).await.is_ok());
+
+	log::info!("Checking if collator-single-core is up");
+	assert!(wait_node_is_up(collator_single_core, 60u64).await.is_ok());
+
+	log::info!("Assigning cores for the parachain");
+	let assign_cores_call = create_assign_core_call(&[(0, PARA_ID_1), (1, PARA_ID_1)]);
+
+	let relay_client: OnlineClient<PolkadotConfig> = alice.wait_client().await?;
+	let res = submit_extrinsic_and_wait_for_finalization_success_with_timeout(
+		&relay_client,
+		&assign_cores_call,
+		&dev::alice(),
+		60u64,
+	)
+	.await;
+	assert!(res.is_ok(), "Extrinsic failed to finalize: {:?}", res.unwrap_err());
+	log::info!("2 more cores assigned to the parachain");
+
+	for (name, block_cnt) in [("collator_single_core", 20.0), ("collator_elastic", 40.0)] {
+		log::info!("Checking block production for {name}");
+		assert!(collator_single_core
```

Comment:
```suggestion
	for (node, block_cnt) in [(collator_single_core, 20.0), (collator_elastic, 40.0)] {
		log::info!("Checking block production for {}", node.name());
		assert!(node
```

Otherwise you are just changing the name but just querying one node 😬 . Same below.

Reply:
Ouch 🙈